Dockerfile

```diff
@@ -117,7 +117,7 @@ RUN curl -L http://http.debian.net/debian/pool/main/libc/libcap2/libcap2_2.22.or
 RUN cd /linux-kernel && \
     make INSTALL_HDR_PATH=/tmp/kheaders headers_install && \
     cd / && \
-    git clone http://git.code.sf.net/p/aufs/aufs-util && \
+    git clone https://github.com/Distrotech/aufs-util.git && \
```

Comment:
I'm honestly very -1 on this change, especially since there's no verification or even hard-lock on a specific commit for this repo.